libs/botan/Makefile

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (C) 2019 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=botan
+PKG_VERSION:=2.9.0
+PKG_RELEASE:=1
+PKG_MAINTAINER:=BangLang Huang <banglang.huang@foxmail.com>
+
+PKG_SOURCE:=$(PKG_VERSION)
```

Comment:
The problem with doing this is that the file will show up as 2.9.0.tar.gz on the download directory. Keep it as $(PKG_NAME)-$(PKG_VERSION)

Reply:
erm no. Add .tar.gz at the end.